### DISCUSSION TITLE
Editor mode on rotation issue || File manager wrong directory?

### DISCUSSION THREAD

**Discussion by plegrand1** [gsantner/markor#1635] [Q&A] [answered]
Hello,
I just discovered Markor and this application is just perfect.
Apparently, the rotation of the smartphone automatically switches to editing mode, is there any way to disable this behavior?

Another thing, maybe a bug, in the file manager, the directory displayed is not the right one, it is the one just above the "real" directory.
For example: 
Te app "says" that i'm in this directory : **/storage/emulated/0/1-Data** but in fact i'm in this directory: **"/storage/emulated/0/1-Data/Markor"**

![image](https://user-images.githubusercontent.com/8543044/158011838-b8775eb0-c93b-4a6d-ba58-157fef3df579.png)
Thanks for your app and your help
Pascal

**Comment by gsantner** [comment, marked as answer]
The ".." is the upper directory, the name of current directoy is in the top Title, which is Markor. So looks all right.

upper ..... /1-Data + current Markor = 1-Data/Markor.. 


#### Rotation
It would be good if you say the app version and other info as well. Bug reports with just one sentence are not helpful to developers.  
Please fill a bug, and remove/edit all rotation text from this discussion, so this dscussion is about the filemanager.

**Reply by plegrand1** [reply]
Hello,
first, thanks for your answer and sorry in advance for my poor english.
Regarding the name of the folder displayed, it is now clearer for me.

I use Markor on a XIAOMI MI 9
Android 11
MIUI 12.5.1
The markor version is the v2.8.6(130)

About "fill a bug", i'm novice with github, can you explain me how to do that ?

My problem is that the rotation of my device switches the application to editor mode regardless of the part used: Files, Todo, QuickNote

Thanks again

**Reply by gsantner** [reply]
the "Issues" tab on this page

**Comment by harshad1** [comment]
I see the editor mode rotation bug. Will add fix.

**Reply by gsantner** [reply]
#1626 #1636 #1635
